components/p3a/brave_p3a_service.cc

```diff
@@ -43,7 +43,7 @@ constexpr char kLastRotationTimeStampPref[] = "p3a.last_rotation_timestamp";
 
 constexpr char kDefaultUploadServerUrl[] = "https://p3a.brave.com/";
 
-constexpr uint64_t kDefaultUploadIntervalSeconds = 60 * 60;  // 1 hour.
+constexpr uint64_t kDefaultUploadIntervalSeconds = 60;  // 1 minute.
```

Comment:
nit: maybe we could do `1 * base::Time::kSecondsPerMinute` so that it's clearer

Reply:
We should transition all of the randomized delays from integral numbers of seconds to base::Time -- that's a more general issue that we should fix across the board.

Reply:
@riastradh-brave probably, `base::TimeDelta`, not `base::Time`? Is there an issue for this?

Reply:
(Sorry, wires crossed -- base::time::kSecondsPerMinute is an integer number of seconds; using base::Time itself is a different issue.  Don't mind me!)